health/micro-ui/web/micro-ui-internals/packages/modules/health-hrms/src/services/service.js

```
@@ -125,23 +125,22 @@ export const formPayloadToUpdateUser = (data, userExisting, tenantId) => {
-      })
-    : [
-        {
-          fromDate: new Date(data?.SelectDateofEmployment).getTime(),
-          toDate: undefined,
-          isCurrentAssignment: true,
-          department: data?.SelectEmployeeDepartment?.code || HRMS_CONSTANTS.DEFAULT_DEPARTMENT,
-          designation: data?.SelectEmployeeDesignation?.code || "undefined",
-        },
-      ];
+  ? userExisting[0].jurisdictions.map((j) => {
+      let jurisdiction = Object.assign({}, j);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'jurisdiction' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)